src/utils/change.ts

```diff
@@ -102,7 +102,7 @@ export function getChangeLogSection(
       .filter((v, i, a) => a.indexOf(v) === i)
       .filter((c) => !c.endsWith('[bot]'));
     if (authors.length > 0) {
-      const contributors = `### ❤️ Thanks to all contributors! ❤️\n\n${authors.join(', ')}`;
+      const contributors = `${config.user.releaseContributorGreeting}\n\n${authors.join(', ')}`;
```

Comment:
#197 uses the plugin settings both options should be consistent as they are pretty similar.

Reply:
I think a user setting fits better here. #197 doesn't use it as it isn't (easily) possible the way the code is composed right now.